### DISCUSSION TITLE
Re: SANA full-rank finetuning/training

### DISCUSSION THREAD

**Discussion by gitanon112** [bghira/SimpleTuner#1252] [Q&A]
Hello,

Thank you for your guide on full-rank training SANA. I just had a couple of questions:
1) Is the training guide still up-to-date/will it work for full-rank training of 2K Sana model?
2) What is the rationale to use SimpleTuner vs SANA native scripts(https://github.com/NVlabs/Sana/blob/main/train_scripts/train.py and https://github.com/NVlabs/Sana/blob/main/train_scripts/train.sh are full-rank training scripts for SANA I believe?). Is SimpleTuner faster?
3) So is the advantage of full-rank training that we can train on large image dataset(like 10k+ images) and we wouldn't be able to do dreambooth training with this many images?
4) If my goal is to make SANA 2k model(which currently struggles with human image generation) capable/better at generating human images, is it as simple as full-rank training on dataset with high-quality human image-text pairs? Or is it more complex than this?

Thanks so much for the help and your work, really appreciate it!

**Comment by bghira** [comment]
Sana is a very disappointing model. I haven't spent a whole lot of time on it. the 2k model might not work at all, who knows.

I never tested their trainer, I'm not aware of any advantages of using either. Both struggle to train Sana due to the model architecture.
